docs/overview/introduction.mdx

```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
 Keep helps with every step of the alert lifecycle:
 1. Creation - Keep offers a framework for creating, debugging, and testing alerts through code that scales with your teams.
 2. Maintenance - Keep integrates with your tools, allowing you to manage all of your alerts within a single interface.
```

Comment:
maybe something else instead of maintenance? sounds weird
```suggestion
1. Maintenance - Keep integrates with your tools, allowing you to manage all of your alerts within a single interface.
```